CheckerboardKarel.java

```diff
@@ -12,5 +12,124 @@
 public class CheckerboardKarel extends SuperKarel {
 
 	// You fill in this part
-
+	public void run(){
+		
+		int n1=8;
+		int n2=8;
```

Comment:
Go and check the problem again.

You are only solving 1 problem i.e. 8x8 checker board. Instead you need to solve generic problem for different size of checker boards as given in the problem.